### DISCUSSION TITLE
Security Onion SOC MFA

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#3803] [Unsupported Versions] [answered]
I was wondering if there were plans to add MFA to the Security Onion SOC, or if there already is a way to do it.

Seeing as it gives you a lot of network information, I think you'd want to protect the SOC as much as possible. I know the firewall is very restrictive, and you need to manually create accounts and so on, but MFA does seem like that extra step of security the SOC could use.

**Comment by weslambert** [comment, marked as answer]
This is on the roadmap.

**Reply by influence2006** [reply]
@weslambert any update on this?

**Reply by dougburks** [reply]
SOC uses Kratos as our authentication backend. Kratos just recently added MFA support. It will take a few months for us to get the necessary plumbing in place.
